<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>se</class>
 <widget class="QWidget" name="se">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_13">
     <property name="title">
      <string>Histogram</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_32">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_31">
        <item row="0" column="2">
         <widget class="QComboBox" name="laserpulses_ComboBox"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>Laser Pulse number:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="laserpulses_display_raw_CheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Display raw</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="PlotWidget" name="laserpulses_PlotWidget"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_14">
     <property name="title">
      <string>Parameters for state estimation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_34">
      <item row="0" column="1">
       <layout class="QGridLayout" name="se_settings_gridLayout">
        <item row="0" column="4">
         <widget class="QPushButton" name="se_setting_add_pushButton">
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="se_setting_comboBox"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="se_method_comboBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="se_method_label">
          <property name="text">
           <string>Estimation method:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="se_setting_delete_pushButton">
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="se_setting_label">
          <property name="text">
           <string>setting</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <spacer name="se_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QGridLayout" name="extraction_param_gridLayout"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>se_method_comboBox</tabstop>
  <tabstop>laserpulses_display_raw_CheckBox</tabstop>
  <tabstop>laserpulses_ComboBox</tabstop>
  <tabstop>laserpulses_PlotWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
